python/example/single_cell_json.py

```diff
@@ -0,0 +1,75 @@
+import arbor
+import pandas
+import seaborn
+import sys
+
+if len(sys.argv) < 4:
+    print("Pass default parameter file, decor file and SWC file to the program")
+    sys.exit(0)
+
+defaults_file = sys.argv[1]
+decor_file    = sys.argv[2]
+swc_file      = sys.argv[3]
+
+# Load inputs from file
+defaults = arbor.load_default_parameters(defaults_file)
+decor    = arbor.load_decor(decor_file)
+morph    = arbor.load_swc_arbor(swc_file)
+
+# Test round trip of default parameter descriptions and decor.
+arbor.store_default_parameters(defaults, "defaults_out.json")
+arbor.store_decor(decor, "decor_out.json")
```

Comment:
This has no effect, as nothing actually is tested. Suggestion: drop this.

Reply:
Maybe I will add the lines to a different example to illustrate the functionality.